extensions/wikia/EditPageLayout/js/editor/WikiaEditor.js

```diff
@@ -1006,7 +1011,9 @@
 		},
 
 		themeLoaded: function() {
-			this.editor.fire('editboxReady', this.editor, $(this.editor.ck.getThemeSpace('contents').$));
+		this.editor.fire('editboxReady', this.editor, $(this.editor.ck.ui.space('contents').$));
+		
+		//this.editor.fire('editboxReady', this.editor, $(this.editor.ck.getThemeSpace( 'contents' ).$));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)

[project-id]: wikia:app